### PR TITLE
Unit test

### PR DESCRIPTION
**Pull request by LivHolm**

### CODE REVIEW

**Comment by Determinatoro** [comment]
quack_api.Test/Resources/Test_recommender should be named TestRecommender

File: quack_api/quack_api.Test/IntegrationTests/IntegrationTests.cs [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 namespace quack_api.Test.IntegrationTests
 {
-    [TestClass]
+    //[TestClass]
```
Comment (Determinatoro) [inline quack_api/quack_api.Test/IntegrationTests/IntegrationTests.cs]: Remove //

File: quack_api/quack_api.Test/UnitTests/RecommenderControllerUnitTests.cs [hunk 2]
```diff
@@ -5,34 +5,43 @@
...
 using quack_api.Enums;
+using Microsoft.Extensions.Configuration;
 
 namespace quack_api.Test.UnitTests
 {
     [TestClass]
     public class RecommenderControllerUnitTests
     {
-        
+        public static IConfiguration InitConfiguration()
```
Comment (Determinatoro) [inline quack_api/quack_api.Test/UnitTests/RecommenderControllerUnitTests.cs]: This method I would move to a utility class called GeneralUtil in a folder called Utilities

File: quack_api/quack_api.Test/UnitTests/RecommenderControllerUnitTests.cs [hunk 3]
```diff
@@ -5,34 +5,43 @@
...
         public async Task RecommenderController_GetPlaylist_Success()
         {
             //Arrange
-            IOptions<TestOptions> testOptions = Options.Create(new TestOptions()
-            {
-                RecommenderPath = @"C:\Users\hliv1\Desktop\Projects\webservice\quack_api\quack_api.Test\Resources\Recommender_Test\src\main.py",
-                PythonPath = @"C:\Users\hliv1\AppData\Local\Programs\Python\Python310\python.exe"
-            });
-            RecommenderController recommenderController = new RecommenderController(testOptions);
+            var config = InitConfiguration();
```
Comment (Determinatoro) [inline quack_api/quack_api.Test/UnitTests/RecommenderControllerUnitTests.cs]: I would add this initializer that runs before each test in order to clean up duplicate code
```
IConfiguration Configuration { get; set } 

[TestInitialize]
public void Setup()
{
 // Runs before each test. (Optional)
 Configuration = InitConfiguration();
}
```

File: quack_api/quack_api/Utilities/CommandLineProcess.cs [hunk 4]
```diff
@@ -4,7 +4,20 @@
 using System.Threading.Tasks;
 
 namespace quack_api.Utilities
-{
+{ 
+    public class PythonPathNotFoundException : Exception
```
Comment (Determinatoro) [inline quack_api/quack_api/Utilities/CommandLineProcess.cs]: I would not use Python in the name so only ```PathNotFoundException``` because the class CommandLineProcess is not only for Python.

**Review by Determinatoro — CHANGES_REQUESTED** [review]
I have left a number of comments

**Review by Determinatoro — APPROVED** [review]
